src/main/java/com/example/todo/demo/users/User.java

```diff
@@ -1,23 +1,22 @@
 package com.example.todo.demo.users;
 
+import lombok.*;
+import lombok.experimental.Accessors;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
-
+import javax.persistence.Entity;
+import javax.persistence.Id;
 import java.util.UUID;
 
-@Data
+@Entity(name = "T_USER")
```

Comment:
Why is it named `T_USER` and not `USER` ?

Reply:
Because the name need to be prefix by T_, if not it will fail to create table (USER is a reserved PSQL key word)